Marlin/src/module/planner.cpp

```diff
@@ -2341,22 +2340,24 @@ bool Planner::_populate_block(block_t * const block, bool split_move,
              *  }
              *  fprintf(stderr, "%.9gf, ", (min + max) / 2);
              */
-            static constexpr int16_t  jd_lut_count = 15;
-            static constexpr uint16_t jd_lut_tll   = 1 << jd_lut_count;
-            static constexpr int16_t  jd_lut_tll0  = __builtin_clz(jd_lut_tll) + 1; // i.e., 16 - jd_lut_count
+            static constexpr int16_t  jd_lut_count = 16;
+            static constexpr uint16_t jd_lut_tll   = _BV(jd_lut_count - 1);
+            static constexpr int16_t  jd_lut_tll0  = __builtin_clz(jd_lut_tll) + 1; // i.e., 16 - jd_lut_count + 1
```

Comment:
Of course, all this rigamarole can be skipped since we know that `__builtin_clz(_BV(n))` will always give the result `15 - n`. We are starting from the point of knowing the bit index, so counting the leading zeros is … indulgent.